man/systemd.unit.xml

```diff
@@ -979,6 +979,16 @@
         ones.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SurviveSoftReboot=</varname></term>
```

Comment:
I think this should be called `IgnoreOnSoftReboot=`, for consistency with the other option. Conceptually, those are similar.